### DISCUSSION TITLE
[SOLVED] Breeze Dark icons look rather rough

### DISCUSSION THREAD

**Discussion by God-damnit-all** [martinrotter/rssguard#594] [General]
I use Fusion's dark theme for what it's worth. I switched to using Breeze Dark since the Breeze icons are very hard to see on the dark theme (for obvious reasons). Honestly I just learned of the option today while looking through the options menu.

![image](https://user-images.githubusercontent.com/53661808/150004295-07b206c6-0661-4e45-8658-c5c318b3fa2d.png)

It's almost like they're not scaling properly, and it looks as though it's purely just a color inversion from Breeze, which makes the anti-aliased edges give it a jagged look rather than a smooth one.

**Comment by martinrotter** [comment]
what operating system?

**Reply by God-damnit-all** [reply]
> what operating system?

Windows 10

**Reply by martinrotter** [reply]
Do you have like very HiDPI display or?

**Reply by God-damnit-all** [reply]
> Do you have like very HiDPI display or?

No, it's just a normal 1080p monitor.

**Reply by martinrotter** [reply]
Thing is, those icons are SVG, they should scale. What happens if you try to change toolbaar icon size to bigger value?

You need latest devbuild for that, test with that.

![image](https://user-images.githubusercontent.com/1255302/150077380-84b1d509-98e3-493e-9b88-d66fda1b0b5f.png)

**Reply by God-damnit-all** [reply]
Hmm, well, scaling them up does make them look better. They just don't seem to look good downscaled.

I did some searching and one trick people do to make svgs render better in Qt is give QPainter a +0.5 offset, as described here: https://github.com/RazrFalcon/resvg/issues/153#issuecomment-509777511

**Reply by martinrotter** [reply]
@ImportTaste Please, report this issue [upstream](https://bugreports.qt.io/secure/Dashboard.jspa). This is out-of-scope for me. I do not want to perform custom-painting in "tool buttons", this should just work. :)
